pages/resources/user-settings.mdx

```diff
@@ -0,0 +1,182 @@
+| inline_attachment_media                        | boolean                                                   | Whether to display attachments when they are uploaded in chat                                              |
+| inline_embed_media                             | boolean                                                   | Whether to display videos and images from links posted in chat                                             |
+| locale                                         | string                                                    | The [language option](/reference#locales) chosen by the user                                               |
+| message_display_compact                        | boolean                                                   | Whether to use the compact Discord display mode                                                            |
+| native_phone_integration_enabled               | boolean                                                   | Whether to enable the new Discord mobile phone number friend requesting feature                            |
+| passwordless                                   | boolean                                                   | Whether to enable passwordless login                                                                       |
+| render_embeds                                  | boolean                                                   | Whether to render message embeds                                                                           |
+| render_reactions                               | boolean                                                   | Whether to render message reactions                                                                        |
+| restricted_guilds                              | array[snowflake]                                          | The IDs of guilds that you will not receive DMs from                                                       |
+| show_current_game                              | boolean                                                   | Whether to display the game that current user is playing                                                   |
+| status                                         | string                                                    | The [status](/resources/presence#status-type) of the user                                                  |
+| stream_notifications_enabled                   | boolean                                                   | Whether to receive stream notifications for friends                                                        |
+| theme                                          | string                                                    | The [theme](/resources/user#client-theme)                                                                  |
```

Comment:
The theme enum should probably be moved to this page (and the links elsewhere updated)